ethcore/sync/src/chain/mod.rs

```diff
@@ -144,8 +143,6 @@ pub const ETH_PROTOCOL_VERSION_62: (u8, u8) = (62, 0x11);
 pub const PAR_PROTOCOL_VERSION_1: (u8, u8) = (1, 0x15);
 /// 2 version of Parity protocol (consensus messages added).
 pub const PAR_PROTOCOL_VERSION_2: (u8, u8) = (2, 0x16);
-/// 3 version of Parity protocol (private transactions messages added).
-pub const PAR_PROTOCOL_VERSION_3: (u8, u8) = (3, 0x18);
```

Comment:
We will never get back to private transactions -> not much sense to even leave this commented out. Just like there's no trace of Tendermint engine left either in this codebase.

Reply:
If we had any proper documentation I would agree because there would be some trace left, because we don't have it this is a good alternative to leave snipped for the future. On the other hand, energy vs benefit for me to further discuss this is very low :)